types/gzip-size/gzip-size-tests.ts

```diff
@@ -1,7 +1,16 @@
-import gzipSize = require('gzip-size');
+import * as fs from 'fs';
+import * as gzipSize from 'gzip-size';
```

Comment:
ES module shouldn't be used here, because `gzipSize` is a function, and An ES6 module namespace object cannot be invoked as a function or with new.

Reply:
@plantain-00 Reverted